Mathlib/SetTheory/Ordinal/CantorNormalForm.lean

```diff
@@ -120,25 +119,27 @@ theorem CNF_lt_snd {b o : Ordinal.{u}} {x : Ordinal × Ordinal} : x ∈ CNF b o
 theorem CNF_snd_lt {b o : Ordinal.{u}} (hb : 1 < b) {x : Ordinal × Ordinal} :
     x ∈ CNF b o → x.2 < b := by
   refine CNFRec b ?_ (fun o ho IH ↦ ?_) o
-  · simp only [CNF_zero, not_mem_nil, IsEmpty.forall_iff]
+  · simp
   · rw [CNF_ne_zero ho]
     intro h
-    cases' (mem_cons.mp h) with h h
-    · rw [h]; simpa only using div_opow_log_lt o hb
+    obtain rfl | h := mem_cons.mp h
+    · exact div_opow_log_lt o hb
     · exact IH h
 
 /-- The exponents of the Cantor normal form are decreasing. -/
 theorem CNF_sorted (b o : Ordinal) : ((CNF b o).map Prod.fst).Sorted (· > ·) := by
   refine CNFRec b ?_ (fun o ho IH ↦ ?_) o
-  · simp only [gt_iff_lt, CNF_zero, map_nil, sorted_nil]
+  · simp
```

Comment:
```suggestion
  · rw [CNF_zero]
    exact sorted_nil
```

This also works, use the one you prefer.

Reply:
This is short enough that I think it's an improvement.